worker/migrations/migrations.go

```diff
@@ -65,6 +74,8 @@ func worker(ctx *job.WorkerContext) error {
 		return migrateToSwiftV3(ctx.Instance.Domain)
 	case swiftV1ToV2:
 		return fmt.Errorf("this migration type is no longer supported")
+	case accountsToOrganization:
+		return migrateAccountsToOrganization(ctx.Instance.Domain)
```

Comment:
Can you update documentation: https://github.com/cozy/cozy-stack/blob/master/docs/workers.md#migrations ?